.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[net]
```

Comment:
(unrelated line)

> Follow-up work to ticket
- Re-enable `publish_passes_error_details_through`
- Drop `History` so we don't need to worry about it ever behaving badly with `multi_thread`